README.md

```diff
@@ -14,10 +14,16 @@ With `yarn`:
 yarn add @front-finance/frontfinance-rn-sdk
 ```
 
-### Getting connection link
+💡 This package requires `react-native-webview` to be installed in your project. Although it is listed as direct dependency, some times it is not installed automatically (This is a known [npm issue](https://stackoverflow.com/questions/18401606/npm-doesnt-install-module-dependencies)). You should install it manually via following command in this case:
+```bash
+npm install --save react-native-webview@11.26.0
 
-The connection link for brokerage connection should be obtained from the [Get catalog link](https://docs.getfront.com/reference/get_api-v1-cataloglink) endpoint. Request must be performed from the server side because it requires the client secret. You will get the response in the following format:
+# or with yarn
+yarn add react-native-webview@11.26.0
+```
 
+### Connect through catalog link `@deprecated (use linkToken instead)`
```

Comment:
Please remove this section from the docs. We want to discourage people from using it.